### PR TITLE
Include off-standard 64 kB limit tweak

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/419/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/420/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -62,6 +62,9 @@ set(UAGENT_CONFIG_SERVER_QUEUE_MAX_SIZE        32000    CACHE STRING "Maximum se
 set(UAGENT_CONFIG_CLIENT_DEAD_TIME             30000    CACHE STRING "Client dead time in milliseconds.")
 set(UAGENT_SERVER_BUFFER_SIZE                  65535    CACHE STRING "Server buffer size.")
 
+# Off-standard features and tweaks
+option(UAGENT_TWEAK_XRCE_WRITE_LIMIT "This feature uses a tweak to allow XRCE WRITE DATA submessages grater than 64 kB." ON)
```
Comment (Acuadros95) [inline CMakeLists.txt:66]:
```suggestion
option(UAGENT_TWEAK_XRCE_WRITE_LIMIT "This feature uses a tweak to allow XRCE WRITE DATA submessages greater than 64 kB." ON)
```

**Review by Acuadros95 — DISMISSED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/420/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/421/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/422/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/423/)

**Review by Acuadros95 — APPROVED** [review]

**Pull request merged**